Tasks/XamarinLicense/XamarinLicense.ps1

```diff
@@ -12,7 +12,7 @@ Write-Verbose "email = $email"
 Write-Verbose "activateAndroid = $activateAndroid"
 Write-Verbose "timeout = $timeout"
 
-$activateAndroidLicense = Convert-String $activateAndroid Boolean
+$activateAndroidLicense = [System.Convert]::ToBoolean($activateAndroid)
```

Comment:
OK. I found the issue. Unfortunately the task.internal module defines a cmdlet named the same thing as the powershell one. I've created a PR which fixes the issue with the Xamarin License task failing: https://github.com/Microsoft/vso-agent-tasks/pull/679